packages/transformers/mdx-2/src/MDXTransformer.js

```diff
@@ -0,0 +1,41 @@
+const {Transformer} = require('@parcel/plugin');
+
+module.exports = new Transformer({
+  async loadConfig({config}) {
+    // config.getConfig only works with CJS
+    let configFile = await config.getConfig(['.mdxrc.js', '.mdxrc.cjs'], {
```

Comment:
To follow https://github.com/mdx-js/mdx/blob/v1/packages/parcel-plugin-mdx/src/MDXAsset.js#L13, I could add `mdx.config.js` and `mdx.config.cjs`.

Or just use those 2 instead of `.mdxrc.js` (as usually the rc files are JSON files, not JS files)